conventions/django.md

```diff
@@ -245,12 +246,32 @@ except Thing.MultipleObjectsReturned:
 
 The second example avoids a race condition and is also more efficient, as only a single database query is required.
 
-## <a name="implicit-ordering">Don't rely on implicit ordering of querysets</a>
+## <a name="implicit-ordering-in-models">Don't add implicit ordering to models</a>
+
+Django supports implicit ordering, where all queries to a model are ordered by one of
+their fields unless specified otherwise:
```

Comment:
this confused me for a while

Reply:
i guess it's because it's implicit to the _caller_, but it's actually explicit in the code example below.

Reply:
maybe we could change it to talking about "default ordering"?

Reply:
I agree that's what it's meant to mean, and I was just copying the language we already use, though in Kraken's source code the concept is referred to as 'default ordering'.

Reply:
Snap! Since this PR is just copying over changes in the private repo, I'll merge this and propose the language change in there first.